pypi_formula_mappings.json

```diff
@@ -222,6 +222,7 @@
   "flintrock": {
     "exclude_packages": ["six"]
   },
+  "fobis": "fobis-py",
```

Comment:
Workaround to avoid adding resource duplicating formula. The PyPI name of package is `FoBiS.py`, but pipgrip outputs `fobis-py` which is seen as a different package:
```console
❯ pipgrip FoBiS.py
fobis-py==3.0.5
configparser==5.3.0
future==0.18.2
```